scripts/test.sh

```diff
@@ -116,15 +116,21 @@ function integration_extra {
 }
 
 function integration_pass {
-  run_for_module "tests" go_test "./integration/..." "parallel" : -timeout="${TIMEOUT:-15m}" "${COMMON_TEST_FLAGS[@]}" "${RUN_ARG[@]}" -p=2 "$@" || return $?
-  run_for_module "tests" go_test "./common/..." "parallel" : --tags=integration -timeout="${TIMEOUT:-15m}" "${COMMON_TEST_FLAGS[@]}" -p=2 "${RUN_ARG[@]}" "$@" || return $?
+  run_for_module "tests" go_test "./integration/..." "parallel" : \
```

Comment:
If this command fails but the following commands are successfully, the L124 return 0. It is not expected. 
Please checkout this https://github.com/etcd-io/etcd/pull/15519#discussion_r1144637951

Reply:
@fuweid I thought the "return $?" below this command would return the AND of exit codes of all commands.

Thanks for the reference link. I shall correct and re-submit a patch.

Reply:
@fuweid Can you please review again?